verify-replace.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+TMP_DIR=/tmp/
+BASE_REPO_PATH=$(mktemp -d ${TMP_DIR}replace-verify.XXX)
+GH_BASE_URL_KS=https://github.com/kubesaw/
+GH_BASE_URL_CRT=https://github.com/codeready-toolchain/
+GH_KSCTL=${GH_BASE_URL_KS}ksctl
+GH_HOST=${GH_BASE_URL_CRT}host-operator
+GH_MEMBER=${GH_BASE_URL_CRT}member-operator
+GH_REGSVC=${GH_BASE_URL_CRT}registration-service
+GH_E2E=${GH_BASE_URL_CRT}toolchain-e2e
+GH_TC=${GH_BASE_URL_CRT}toolchain-common
+C_PATH=${PWD}
+ERRORLIST=()
+
+echo Initiating verify-replace on dependent repos
+for repo in ${GH_HOST} ${GH_REGSVC} ${GH_KSCTL} ${GH_MEMBER} ${GH_E2E}
+do
+    echo =========================================================================================
+    echo  
+    echo $(basename $repo)
```

Comment:
My knowledge of bashscript is limited, so i don't know if this is helpful, but I see a warning in my IDE for this line:
"Quote this to prevent word splitting."

Reply:
added quotes